src/frontend/komponenter/Fagsak/FagsakContainer.tsx

```diff
@@ -16,11 +15,6 @@ import JournalpostListe from './journalposter/JournalpostListe';
 import Personlinje from './Personlinje/Personlinje';
 import Saksoversikt from './Saksoversikt/Saksoversikt';
 
-const Hovedinnhold = styled.div`
```

Comment:
Har du testet at høyde og scroll funker og ser bra ut på de andre sidene som var omfattet av denne? Husker at det var noe scrollproblematikk som førte til at jeg la den akkurat her 🤔

Reply:
Kan være vi må sette høyde eksplisitt både her og i BehandlingContainer for å sikre scroll på alle sidene

Reply:
Sweet! Testet selv, funker bra!